cast/src/lib.rs

```diff
@@ -880,7 +933,7 @@ impl SimpleCast {
         let code = meta.source_code();
 
         if code.is_empty() {
-            return Err(eyre::eyre!("unverified contract"))
+            return Err(eyre::eyre!("unverified contract"));
```

Comment:
```suggestion
            return Err(eyre::eyre!("unverified contract"))
```

Reply:
there is `eyre::bail!` which does exactly that, I think